xml/ay_bigfile.xml

```diff
@@ -4489,6 +4489,36 @@ config:type="boolean"&gt;true&lt;/diag&gt;</screen>
     <remark role="fixme">Add a short description</remark>
    </para>
 
+   <sect2 xml:id="Software.Selections.base_product">
+    <title>Product Selection</title>
+    <para>
+     Starting with SLE-15 all products are distributed using one media. You have to
+     choose which product you wish to install. You can do it explicitly using.
+     <literal>product</literal> option
+     <example>
+       <title>Explicit product selection</title>
+       <para>
+         &lt;software&gt;&lt;products&gt;&lt;product&gt;SLED15&lt;/product&gt;&lt;/products&gt;&lt;/software&gt;
```

Comment:
No :) Whitespace is a tricky part of XML. If all content in a tag is inside other tags, then the whitespace is stripped and it is ok to include it. 
```xml
&lt;software&gt;
  &lt;products config:type="list"&gt;
    &lt;product&gt;SLED15&lt;/product&gt;
  &lt;/products&gt;
&lt;/software&gt;
```

 See the next section for an example, https://github.com/SUSE/doc-sle/pull/167/files#diff-8640e213b28bf6344ba837bf30c6c0adR4529

Reply:
agh, didn't know thanks

Reply:
@mvidner Whitespace handling is indeed a bit tricky but in a slightly different way: The whitespace handling actually heavily depends on your DTD/Schema. In DocBook, there are a couple elements (`<screen/>`, <programlisting/>`, ...) that are defined as "verbatim elements" (i.e. elements where whitespace should be kept as-is). All other elements are treated like "normal" XML, which as an sed pattern would ~be: `s/[ \n\r]+/ /` (i.e. only the first space is taken into account, all others are removed)